pkg/math/voteweighted/voteweighted.go

```diff
@@ -160,6 +105,76 @@ func Median(
 	}
 }
 
+func PriceInfoFromProviders(
```

Comment:
not: this should be `PriceInfoFromValidatorStake`, or similar, not sure why the notion of a `Provider` is involved here